### PR TITLE
feat: max slippage 20%

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

File: reference/Kyber-Swap-Aggregator-APIs.yaml [hunk 1]
```diff
@@ -172,6 +173,11 @@ paths:
           in: query
           name: clientData
           description: Json string present your client
+        - schema:
+            type: string
+          in: header
+          name: Accept-Version
+          description: Latest
```
Comment (tiendv89) [inline reference/Kyber-Swap-Aggregator-APIs.yaml:180]: sao lại description ? value hay gì chứ

Comment (nguyenhuudungz) [reply reference/Kyber-Swap-Aggregator-APIs.yaml]: Fixed.

File: reference/Kyber-Swap-Aggregator-APIs.yaml [hunk 2]
```diff
@@ -172,17 +173,27 @@ paths:
...
+        - schema:
+            type: string
+            enum:
+              - Latest
+          in: header
+          name: Accept-Version
+          description: |
+            Use latest version or release candidate. \
+            Empty means release candidate version. \
+            `Latest` means latest version.
```
Comment (tiendv89) [inline reference/Kyber-Swap-Aggregator-APIs.yaml:185]: emphasize that you should have this value as LATEST. we require them as so

Comment (nguyenhuudungz) [reply reference/Kyber-Swap-Aggregator-APIs.yaml]: Fixed.

File: reference/Kyber-Swap-Aggregator-APIs.yaml [hunk 3]
```diff
@@ -172,17 +173,25 @@ paths:
           in: query
           name: clientData
           description: Json string present your client
+        - schema:
+            type: string
+            enum:
+              - Latest
+          in: header
+          name: Accept-Version
+          description: Version of API
```
Comment (tiendv89) [inline reference/Kyber-Swap-Aggregator-APIs.yaml:182]: Không phải version of API, description của nó là `indicates latest version of aggregator`

**Review by tiendv89 — APPROVED** [review]

**Pull request merged**